### ISSUE TITLE
Enable support for running custom init containers

### ISSUE THREAD

**Issue by nuwang** [CloudVE/galaxy-helm#30]
We should provide a mechanism for running custom init scripts like the postgres chart does. These scripts would provide a hook to perform such tasks as installing additional tools on startup, downloading remote data or whatever else a user may wish to do

**Comment by almahmoud** [comment]
Just to be clear, do we want init `sh` scripts put in as configmaps or customizable init containers with a custom image and command?

**Comment by pcm32** [comment]
Yes, we rely on this kind of functionality on our v2 setup. Current approach in v2 is that startup script will try to start a `post-start-actions.sh` script expected in a defined location. The admin can write whatever they want to that `post-start-actions.sh`. This is meant currently at the other side for operations that require a running galaxy instance. We use it in particular to install workflows and shared them (through ephemeris I think).
